src/main/java/lambda/practice/service/operations/CalculatorOperations.java

```diff
@@ -0,0 +1,32 @@
+package lambda.practice.service.operations;
+
+public class CalculatorOperations implements CalculatorOperationsInterface {
+    @Override
+    public double add(double firstValue, double secondValue) {
+        return firstValue + secondValue;
+    }
+
+    @Override
+    public double subtract(double firstValue, double secondValue) {
+        return firstValue - secondValue;
+    }
+
+    @Override
+    public double multiply(double firstValue, double secondValue) {
+        return firstValue * secondValue;
+    }
+
+    @Override
+    public double divide(double firstValue, double secondValue) {
+        try {
+            return firstValue / secondValue;
+        } catch (ArithmeticException e) {
+            throw new ArithmeticException();
+        }
```

Comment:
What effect will this ```try-catch``` block have if ```secondValue``` will be 0?

Reply:
Thanks for your advice, done.